aws-android-sdk-appsync-gradle-plugin/src/main/groovy/com/apollographql/apollo/gradle/ApolloPlugin.groovy

```diff
@@ -148,7 +148,7 @@ class ApolloPlugin implements Plugin<Project> {
             description = "Generate an IR file using apollo-codegen for ${sourceSetOrVariantName.capitalize()} GraphQL queries"
             dependsOn(ApolloCodeGenInstallTask.NAME)
             sourceSets.each { sourceSet ->
-                inputs.file(sourceSet.graphql).skipWhenEmpty()
+                inputs.files(sourceSet.graphql).skipWhenEmpty()
```

Comment:
Is this the only change required to make the gradle plugin compatible with Gradle 5? How was this determined?